src/Splat.Drawing.Tests/API/ApiApprovalTests.cs

```diff
@@ -4,23 +4,25 @@
 // See the LICENSE file in the project root for full license information.
 
 using System.Diagnostics.CodeAnalysis;
+using System.Threading.Tasks;
+using VerifyXunit;
 using Xunit;
 
+#pragma warning disable SA1615 // Element return value should be documented
```

Comment:
I would prefer this to be suppressed in test projects. let me know if you want a PR